challenge_7/python/sarcodian/src/challenge_7.py

```diff
@@ -0,0 +1,12 @@
+def missing_int(L):
+
+    sum_n = 0
+    sum_n_less_1 = 0
+
+    for i in range(len(L)+1):
+        sum_n += i
```

Comment:
As sent to you in slack, here's a great video on the topic (and also for the reference of others)
https://www.youtube.com/watch?v=ZoWgzG_r2qo